### PR TITLE
DOC: [META-372] Adding Metadata Store logging documentation

### PR DESCRIPTION
**Pull request by xtreme-jason-smith**

### CODE REVIEW

File: install-components.md [hunk 1]
```diff
@@ -1570,6 +1570,7 @@ To install Supply Chain Security Tools - Store:
       storage_class_name                                     string   The storage class name of the persistent volume used by Postgres database for storing data. The default value will use the default class name defined on the cluster.
       database_request_storage          10Gi                 string   The storage requested of the persistent volume used by Postgres database for storing data.
       add_default_rw_service_account    true                 string   Adds a read-write service account which can be used to obtain access token to use metadata-store CLI
+      log_level                         default              string   Sets the log level. This can be set to "minimum", "less", "default", "more", "debug" or "trace". "minimum" currently does not output logs. "less" outputs log configuration options only. "default" and "more" outputs API endpoint access information. "debug" and "trade" outputs extended API endpoint access information(such as body payload) and other debug information.
```
Comment (xtreme-debbie-chen) [inline install-components.md:1573]: `"debug" and "trade"` - I assume "trade" is meant to be "trace"?

File: scst-store/logs.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+This is done to ensure two things:
+1. The application interprets the values of the query/path parameters correctly
+2. Help figure out which log lines are associated with a particular API request. Since there can be a number of simultaneous endpoint calls, this is a first attempt at trying to group logs by specific calls. An improved way to identify and group log by specific request is in the work for a future release.
+
+##### API Payload Log Output
+As mentioned at the start of this section, setting the log level to `debug` will log body payload data for both the request and response of an API call.
+
+Using `debug` log level, over `default`, for this information was done for the following two reasons:
+* Body payloads can be very large as they contain full cycloneDX and SBOM information. By moving the payload information at this level, it helps keep the production log output to a reasonable size.
```
Comment (xtreme-debbie-chen) [inline scst-store/logs.md:93]: `cycloneDX` => `CycloneDX`

**Comment by RichardJJG** [comment]
Is this ready to merge?

**Comment by jncd** [comment]
@xtreme-jason-smith and @xtreme-debbie-chen 
Just following up on Richard's question: When this is ready to merge, please resolve conflicts and let us know.
Thanks!

**Comment by jncd** [comment]
Thanks @xtreme-jason-smith 
I've merged this now. Probably I'll assign @RobC55 to do the review, since he's the primary writer for scst-store.

**Comment by xtreme-jason-smith** [comment]
Sorry for the late reply, but this MR wasn't ready to be merged in. There were still corrections to be made. I will rebase our fork and do another MR with the corrections.

**Comment by jncd** [comment]
Hi @xtreme-jason-smith 
Oops, going forward, please leave a PR in draft status and note clearly in the message if a PR is not ready to be merged (and if not ready, when you expect it to be ready). 

So, where do we go from here? 
Do you want to leave what you have in the doc? Or, are you going to make another PR to pull out these merged changes?

**Comment by xtreme-jason-smith** [comment]
Hi @jncd ,

I will remember that for next time, thanks. I think I created that MR by accident too quickly as well, which caused part of the confusion as well.

Leave it as is. I created a follow up PR: https://github.com/pivotal/docs-tap/pull/558 which includes all the updated information. It looks like it got merged in.

So, what is in the current main branch of this repo is all up-to-date.

**Comment by jncd** [comment]
Thanks @xtreme-jason-smith 
Sounds good. I've merged your latest and also pushed to the staging site. You can check it out for formatting and stuff here: https://docs-staging.vmware.com/en/Tanzu-Application-Platform/0.4/tap/GUID-overview.html

**Comment by xtreme-jason-smith** [comment]
Perfect, thank you @jncd !

**Pull request merged**